### DISCUSSION TITLE
Mod selection personal presets - additive/exclusive presets setting

### DISCUSSION THREAD

**Discussion by zavoxd** [ppy/osu#30364] [Ideas]
Hi!
The personal presets in mod selection is a huge game changer for me. I have one idea how to improve it.

My idea is to have a button here somewhere:
![image](https://github.com/user-attachments/assets/4d958a96-49df-4623-8246-d563a5219b3d)
This button would switch the mod preset to be additive or exclusive. What I mean is that additive presets would add the mods on top of the already selected ones and exclusive presets would overwrite all mods to match the preset (like it works now by default).

Example from myself: I really hate high AR, so I often play maps with difficulty adjust setting the AR to 9. But when I have NF or HD enabled (which is often), clicking the AR9 preset disables those mods and that's a little annoyance for me in this case.
If the preset would turn on AR9 and also keep other mods enabled, that would be pretty awesome.

I hope I explained it well :D

**Comment by bdach** [comment]
Sounds much too complicated to me. I wouldn't want to see this sort of thing in.

**Reply by zavoxd** [reply]
I don't see how it would be complicated. It could be so by default it works as it is right now and only if you don't like that default mod overwriting behavior you could turn this setting/button/tickbox on. I believe that it could be done with only a little amount of code and I think there will definitely be some players that will appreciate this feature.
